portal-api/src/v1/api.js

```diff
@@ -450,6 +450,27 @@ const getNextReportPeriodByBankId = async (bankId) => {
   }
 };
 
+/**
+ * Gets the portal facility amendment
+ * @param {string} facilityId - id of the facility to amend
+ * @param {string} amendmentId - id of the facility amendment
+ * @returns {Promise<(import('@ukef/dtfs2-common').FacilityAmendmentWithUkefId)>} - the amendment
+ */
+const getPortalFacilityAmendment = async (facilityId, amendmentId) => {
+  try {
+    const response = await axios({
+      method: 'get',
+      url: `${DTFS_CENTRAL_API_URL}/v1/portal/facilities/${facilityId}/amendments/${amendmentId}`,
+      headers: headers.central,
+    });
+
+    return response.data;
+  } catch (error) {
+    console.error(error);
```

Comment:
could we add a log as well for the error? eg Error getting portal facility amendment

Reply:
Good spot, now done - thanks!